packages/pipeline-editor/src/PropertiesPanel/properties-utils.ts

```diff
@@ -164,8 +164,8 @@ const properties: CommonPropertiesSchema = {
         label: {
           default: "File Dependencies",
         },
-        // control: "custom",
```

Comment:
you shouldn't have to change anything here, I just left it for reference of what the common properties schema looks like